src/index.js

```diff
@@ -22,3 +24,24 @@ const server = new ApolloServer({
 server.listen({ port: process.env.PORT }).then(() => {
   console.log(`🚀 Server ready at ${process.env.PORT}`);
 });
+
+// Connect Sequelize to Postgres database
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize("alexmerecka", "alexmerecka", "password", {
+  host: "localhost",
+  dialect: "postgres",
+  pool: {
+    max: 9,
+    min: 0,
+    idle: 10000
+  }
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log("Success!");
```

Comment:
can we make this message more clear? right now if i were reading the logs i'd just see a random success. what succeeded?

Reply:
Updated to better message.